src/pudl/workspace/datastore.py

```diff
@@ -268,8 +268,12 @@ def __init__(
             self._cache.add_cache_layer(
                 resource_cache.LocalFileCache(local_cache_path))
         if gcs_cache_path:
-            self._cache.add_cache_layer(
-                resource_cache.GoogleCloudStorageCache(gcs_cache_path))
```

Comment:
Perhaps this would be best solved by creating generic `RemoteCache` that uses `fsspec` under the hood?